app/waf/src/models/monitor.model.ts

```diff
@@ -0,0 +1,46 @@
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class Monitor extends Entity {
+  @property({
+    type: 'string',
+    id: true,
+    required: true,
+  })
+  id: string;
+
+  @property({
+    type: 'number',
+    default: 5,
+  })
+  interval?: number;
```

Comment:
If a property always have a default value, we may remove '?', so that the var type will be a little simpler.